packages/ui/components/ui/button.tsx

```diff
@@ -4,7 +4,7 @@ import { cva, type VariantProps } from 'class-variance-authority';
 import { cn } from '../../lib/utils';
 
 const buttonVariants = cva(
-  'inline-flex items-center justify-center rounded-lg font-bold ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50',
+  'inline-flex items-center justify-center rounded-lg px-4 font-bold ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50',
```

Comment:
Finally, we no longer need to add padding to every button, lol

Reply:
Does that mean we have to remove padding from all our buttons to coincide with this change?

Reply:
I'd like to eventually, but it doesn't harm anything to leave them as-is for now, since they'll override the default padding.